src/nvim/edit.c

```diff
@@ -2899,7 +2900,7 @@ static void ins_compl_dictionaries(char_u *dict_start, char_u *pat, int flags, i
   char_u *ptr;
   char_u *buf;
   regmatch_T regmatch;
-  char_u **files;
+  char_u * *files;
```

Comment:
```suggestion
  char_u **files;
```

These still need changing.

Reply:
Done.

Reply:
The others also need fixing.